python/tk_desktop/desktop_engine_project_implementation.py

```diff
@@ -251,9 +251,9 @@ def _initialize_application(self):
         # Make the name pretty for the tray and the task manager.
         app.setApplicationName("%s Python" % self._engine.context.project["name"])
         # set default icon
-        python_icon = os.path.realpath(os.path.join(
-            os.path.dirname(__file__),
-            "..", "..", "resources", "python_icon.png"))
+        python_icon = os.path.join(
+            self._engine.disk_location, "icon_bg_256.png"
```

Comment:
minor: suggest leaving the icon name as `python_python.png` or maybe `icon_python_proc.png`.